dashboard/static/js/d3.tip.v0.6.3.js

```diff
@@ -0,0 +1,280 @@
+// d3.tip
```

Comment:
Can we pull this from npm ? https://www.npmjs.com/package/d3-tip 

Maybe need the bower -> npm patch that I have like 90% done ;)

Reply:
Yes... I agree Once you have it running I will add to `package.json`